.evergreen/tasks.yml

```diff
@@ -275,7 +275,7 @@ tasks:
           target_platform: '--platform=darwin'
 
   # copied as test-packaged-app-macos due to depends_on variation
-
+  
```

Comment:
NIT: what's adding this space?

Reply:
This file is built automatically, not sure why on my machine it adds these spaces.